contactform.php

```diff
@@ -41,29 +53,187 @@ public function __construct()
         $this->author = 'PrestaShop';
         $this->tab = 'front_office_features';
         $this->version = '3.0.0';
-
         $this->bootstrap = true;
+
         parent::__construct();
 
         $this->displayName = $this->trans('Contact form', array(), 'Modules.Contactform.Admin');
         $this->description = $this->trans('Adds a contact form to the "Contact us" page.', array(), 'Modules.Contactform.Admin');
         $this->ps_versions_compliancy = array('min' => '1.7.2.0', 'max' => _PS_VERSION_);
     }
 
+    /**
+     * @return bool
+     */
     public function install()
     {
         return parent::install();
     }
 
+    /**
+     * @return string
+     */
+    public function getContent()
+    {
+
+        $message = $this->trans(
+            'For even more security on your website forms, consult our Security & Access modules category on the %s',
+            array($this->getSecurityMarketPlaceLink()),
```

Comment:
Don't you want to add placeholders like in the core? Like `%link%`

Reply:
totally, thanks :)